int64 player_id
int64 action
---
bool success
